pkg/migrate/azure/dns.go

```diff
@@ -0,0 +1,1335 @@
+package azure
```

Comment:
i would like this package to be `pkg/migrate/azure/privatedns`

Reply:
Fixed.